account_tax_balance/migrations/14.0.1.0.0/pre-migration.py

```diff
@@ -1,17 +1,26 @@
 # Copyright 2020 Ozono Multimedia S.L.L.
+# Copyright 2021 Simone Rubino - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 from openupgradelib import openupgrade
 
-field_renames = [
-    (
-        "account.move",
-        "account_move",
-        "move_type",
-        "financial_type",
-    ),
-]
-
 
 @openupgrade.migrate()
 def migrate(env, version):
-    openupgrade.rename_fields(env, field_renames)
+    move_table_name = "account_move"
+    enterprise_move_type_rename = "move_type_custom"
+    if openupgrade.column_exists(env.cr, move_table_name, enterprise_move_type_rename):
+        old_move_type_col = enterprise_move_type_rename
+    else:
+        old_move_type_col = openupgrade.get_legacy_name("move_type")
+
+    openupgrade.rename_fields(
```

Comment:
This doesn't serve for enterprise upgrade, as I don't think filters and the rest of the stuff is renamed from `move_type` to `move_type_custom`.

Reply:
Thanks, I've separated the two approaches, I'll remove the OpenUpgrade branch depending on how https://github.com/OCA/account-financial-reporting/pull/771#discussion_r614902420 goes